pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -172,5 +172,5 @@ def country_full_name
 end
 
 def total_hoc_count
-  all_unique_events.count
+  DB[:forms].where(kind: 'HocSignup2015').group(:email, :name).select(:email, :name).count
```

Comment:
removing the explicit `select(:email, :name)` clause still works but no longer makes use of the index, so it would be like the 0.77sec query above instead of the 0.20 sec one.